### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by trevormunoz** [TEIC/CETEIcean#6]
Following the instructions in the README, I ran: `npm i` and then: `npm run build`. Instead of a `CETEI.js` file I received the following error:

```
› npm run build

> CETEIcean@0.0.0 build /Users/umd-laptop/Code/CETEIcean
> rollup -c rollup.config.js

Error loading /Users/umd-laptop/Code/CETEIcean/src/CETEI.js: It looks like your Babel configuration specifies a module transformer. Please disable it. If you're using the "es2015" preset, consider using "es2015-rollup" instead. See https://github.com/rollup/rollup-plugin-babel#configuring-babel for more information
Error: Error loading /Users/umd-laptop/Code/CETEIcean/src/CETEI.js: It looks like your Babel configuration specifies a module transformer. Please disable it. If you're using the "es2015" preset, consider using "es2015-rollup" instead. See https://github.com/rollup/rollup-plugin-babel#configuring-babel for more information
    at preflightCheck (/Users/umd-laptop/Code/CETEIcean/node_modules/rollup-plugin-babel/dist/rollup-plugin-babel.cjs.js:43:102)
    at transform$1 (/Users/umd-laptop/Code/CETEIcean/node_modules/rollup-plugin-babel/dist/rollup-plugin-babel.cjs.js:104:18)
    at /Users/umd-laptop/Code/CETEIcean/node_modules/rollup/src/utils/transform.js:18:28
    at tryCatch (/Users/umd-laptop/Code/CETEIcean/node_modules/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:187:12)
    at invokeCallback (/Users/umd-laptop/Code/CETEIcean/node_modules/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:199:13)
    at /Users/umd-laptop/Code/CETEIcean/node_modules/rollup/node_modules/es6-promise/lib/es6-promise/then.js:27:7
    at flush (/Users/umd-laptop/Code/CETEIcean/node_modules/rollup/node_modules/es6-promise/lib/es6-promise/asap.js:87:5)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
    at Module.runMain (module.js:577:11)
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.3.1/bin/node" "/Users/umd-laptop/.node/bin/npm" "run" "build"
npm ERR! node v6.3.1
npm ERR! npm  v3.9.2
npm ERR! code ELIFECYCLE
npm ERR! CETEIcean@0.0.0 build: `rollup -c rollup.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the CETEIcean@0.0.0 build script 'rollup -c rollup.config.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the CETEIcean package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rollup -c rollup.config.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs CETEIcean
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls CETEIcean
npm ERR! There is likely additional logging output above.
```

node version 6.3.1
npm version 3.9.2

**Comment by hcayless** [comment]
I think this may be an upstream issue with babel/rollup. Looks like there's work happening to resolve it (see https://github.com/rollup/rollup-plugin-babel/issues/72). Think you just picked a bad time to install, @trevormunoz.

**Comment by hcayless** [comment]
Was just able to install and build successfully, so I think this is now resolved with 054e258543819a79250093f296341cbf668a6a7d. Make sure you git pull before trying again.

**Comment by trevormunoz** [comment]
Yep, confirming this builds successfully now
